modules/misunderstood/src/views/full/MainScreen/ResolvedEventsList.tsx

```diff
@@ -37,9 +37,9 @@ const ResolvedEventsList = ({ events, resetEvent }: Props) =>
             </td>
             <td>{event.updatedAt}</td>
             <td>
-              <Button onClick={() => resetEvent('' + event.id)} small icon="refresh" intent={Intent.PRIMARY}>
+              {resetEvent && (<Button onClick={() => resetEvent(`${event.id}`)} small icon="refresh" intent={Intent.PRIMARY}>
```

Comment:
I don't think string literals are necessary there... if it's to make it as a string, why not fix the typings of the parent method?

Reply:
Good point! I fixed the typing issues related to the changes I made. I'll fix the rest in another PR!